### ISSUE TITLE
TypeError at /api/trainfaces/  Object of type function is not JSON serializable

### ISSUE THREAD

**Issue by alexislefebvre** [LibrePhotos/librephotos#162]
I installed LibrePhotos with default config and some differences:
- folders are mounted through Samba on my host
- I added the `:ro` flag so that the original photos are on `read-only` mode

The scanning is running.

I get a 500 code when I click on “Train Faces”:

<details><summary>See traceback</summary>

```
Environment:


Request Method: GET
Request URL: http://backend/api/trainfaces/

Django Version: 3.0.10
Python Version: 3.8.5
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.postgres',
 'api',
 'nextcloud',
 'rest_framework',
 'corsheaders',
 'django_extensions',
 'django_rq',
 'constance',
 'constance.backends.database']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'corsheaders.middleware.CorsMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'api.middleware.FingerPrintMiddleware']



Traceback (most recent call last):
  File "/miniconda/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/miniconda/lib/python3.8/site-packages/django/core/handlers/base.py", line 145, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/miniconda/lib/python3.8/site-packages/django/core/handlers/base.py", line 143, in _get_response
    response = response.render()
  File "/miniconda/lib/python3.8/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/miniconda/lib/python3.8/site-packages/rest_framework/response.py", line 70, in rendered_content
    ret = renderer.render(self.data, accepted_media_type, context)
  File "/miniconda/lib/python3.8/site-packages/rest_framework/renderers.py", line 100, in render
    ret = json.dumps(
  File "/miniconda/lib/python3.8/site-packages/rest_framework/utils/json.py", line 25, in dumps
    return json.dumps(*args, **kwargs)
  File "/miniconda/lib/python3.8/json/__init__.py", line 234, in dumps
    return cls(
  File "/miniconda/lib/python3.8/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/miniconda/lib/python3.8/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/miniconda/lib/python3.8/site-packages/rest_framework/utils/encoders.py", line 67, in default
    return super().default(obj)
  File "/miniconda/lib/python3.8/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '

Exception Type: TypeError at /api/trainfaces/
Exception Value: Object of type function is not JSON serializable
```

<details>

**Comment by derneuere** [comment]
I fixed #163 and could not reproduce your issue after that. It is probably related to #163 and should work in half an hour :)

**Comment by alexislefebvre** [comment]
The issue is solved, thanks.
